tests/results/ngf-upgrade/edge/edge-plus.md

```diff
@@ -6,46 +6,48 @@ NGINX Plus: true
 
 NGINX Gateway Fabric:
 
-- Commit: e7d217a8f01fb3c8fc4507ef6f0e7feead667f20
-- Date: 2024-11-14T18:42:55Z
+- Commit: f7722f679cb574e672f367c5def3d273c13ce6e0
+- Date: 2025-01-01T02:08:05Z
 - Dirty: false
 
 GKE Cluster:
 
 - Node count: 12
-- k8s version: v1.30.5-gke.1443001
+- k8s version: v1.30.6-gke.1125000
 - vCPUs per node: 16
-- RAM per node: 65853972Ki
+- RAM per node: 65853984Ki
 - Max pods per node: 110
 - Zone: us-west1-b
 - Instance Type: n2d-standard-16
 
-## Test: Send http /coffee traffic
+## Test: Send https /tea traffic
 
 ```text
-Requests      [total, rate, throughput]         6000, 100.02, 100.02
-Duration      [total, attack, wait]             59.991s, 59.99s, 772.721µs
-Latencies     [min, mean, 50, 90, 95, 99, max]  596.14µs, 835.746µs, 798.892µs, 926.941µs, 975.553µs, 1.151ms, 14.047ms
-Bytes In      [total, mean]                     956060, 159.34
+Requests      [total, rate, throughput]         6000, 100.02, 58.52
+Duration      [total, attack, wait]             59.993s, 59.991s, 2.514ms
+Latencies     [min, mean, 50, 90, 95, 99, max]  703.663µs, 1.622ms, 966.753µs, 2.707ms, 2.807ms, 3.282ms, 16.336ms
+Bytes In      [total, mean]                     1021816, 170.30
 Bytes Out     [total, mean]                     0, 0.00
-Success       [ratio]                           100.00%
-Status Codes  [code:count]                      200:6000  
+Success       [ratio]                           58.52%
+Status Codes  [code:count]                      200:3511  503:2489  
```

Comment:
That's concerning. Looking at the graph, makes me think that traffic is completely failing after an upgrade. Probably related to our good friend R33.

Reply:
I'll take a look at this later today.

Reply:
https://github.com/nginxinc/nginx-gateway-fabric/pull/2967